app/webpacker/styles/components/eventbox.scss

```diff
@@ -0,0 +1,60 @@
+
+.event-resultbox {
```

Comment:
Looks like there's a fair amount of duplication with eventboxshort, it may be cleaner as a modifier just overwriting the different attributes? `eventbox--short`

Reply:
yep, absolutely, but this design is to be replaced so was just a quick hack.
if it remains ill change it